templates/user/base-users-table.html

```diff
@@ -2,9 +2,12 @@
 <tr>
     <th class="header rank">{{ rank_header or _("Rank") }}</th>
     {% block after_rank_head %}{% endblock %}
-    <th class="header username">{{ _('Username') }}</th>
+    <th class="header username">
+        {% if sort_links %}<a href="{{ sort_links.user__username }}">{% endif %}
```

Comment:
Will this cause colours to change unexpectedly?

Reply:
It doesn't seem to when I test it.
![image](https://user-images.githubusercontent.com/31290848/93167688-26fdb800-f6ef-11ea-8c11-17421ca187dd.png)